virt/storage/virt-configuring-storage-profile.adoc

```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You must configure storage profiles if your storage provider is not recognized by the Containerized Data Importer (CDI)
+You must configure storage profiles if your storage provider is not recognized by the Containerized Data Importer (CDI).
```

Comment:
I would suggest rewriting to active voice: "You must configure storage profiles if the Containerized Data Importer (CDI) does not recognize your storage provider." Or possibly even: "If the Containerized Data Importer (CDI) does not recognize your storage provider, you must configure storage profiles".